client/app/account/user/security/users/add/user-users-add.html

```diff
@@ -6,6 +6,13 @@
 
         <div data-wizard-step
              data-wizard-step-valid="$ctrl.userAccountUsersAddForm.$valid">
+            <p class="m-0"
+               data-translate="user_users_info"
+               data-translate-values="{customerCode: $ctrl.me.customerCode}">
+            </p>
+
+            <br/>
```

Comment:
Avoid using `<br/>` tag.

You can simply use the default margin of the upper `<p>` element like this:

```html
<p data-translate="user_users_info"
   data-translate-values="{customerCode: $ctrl.me.customerCode}">
</p>
```

Reply:
Done